app/controllers/users/sessions_controller.rb

```diff
@@ -0,0 +1,3 @@
+class Users::SessionsController < Devise::SessionsController
+  after_action :verify_authorized, except: [:new, :create, :destroy]
```

Comment:
Why are we not verifying authorization after a new session has been created or destroyed? Is that because we're checking in the view as to whether or not they're a teacher? @jedschneider